lib/season_stats_module.rb

```diff
@@ -39,33 +39,43 @@ def worst_coach(season_id)
     end[0]
   end
 
-  def most_accurate_team(season_id)
-    team_hash = Hash.new
-    season_teams(season_id).each do |team|
-      total_shots = game_teams_data_for_season(season_id).sum do |game|
-        if game.team_id == team
-          game.shots
-        else
-          0
-        end
+  def total_shots_by_team(season_id, team)
```

Comment:
Switching this from a hash to an array was a good call. It makes it easier to follow.